packages/fast-components-react-base/src/card/card.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import Foundation, { HandledProps } from "../foundation";
+import {
+    CardHTMLTags,
+    ICardHandledProps,
+    ICardManagedClasses,
+    ICardUnhandledProps
+} from "./card.props";
+import { ICardClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+
+/* tslint:disable-next-line */
+class Card extends Foundation<ICardHandledProps & ICardManagedClasses,  ICardUnhandledProps, {}> {
+    public static displayName: string = "Card";
```

Comment:
Should the public property for `displayName` be `readonly` of the class?

Reply:
I don't think it's necessary, but @nicholasrice can chime in. This matches what we are doing elsewhere, so I would prefer to make that change (if we decide to) in a new PR.

Reply:
I don't see a reason for it to be `readonly`. If folks want to re-assign it for some reason it doesn't matter to us. For instance, if they already have a `Card` component they may wan't to rename this `FASTCard` for debugging purposes. Probably unlikely, but I don't see a reason to stop them.

Reply:
That's seems like a valid use case.  disregard.